cache/fifo_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package cache
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFIFOCache(t *testing.T) {
+	limit := 10
```

Comment:
Maybe limit should be a parameter in the test table?

Reply:
Good point!

Reply:
Resolved

Reply:
thanks!